omod/src/main/java/org/openmrs/module/spa/SpaActivator.java

```diff
@@ -11,19 +11,57 @@
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
 import org.openmrs.module.BaseModuleActivator;
+import org.openmrs.module.ModuleConstants;
+import org.openmrs.module.ModuleException;
+import org.openmrs.util.OpenmrsUtil;
+
+import java.io.File;
 
 public class SpaActivator extends BaseModuleActivator {
 
 	private final Log log = LogFactory.getLog(getClass());
+	private final String FRONTENT_DIRECTORY = "frontend";
+	private final String SINGLE_SPA_STATIC_FILES_DIR = "spa.frontend.directory";
 	
 	@Override
 	public void started() {
+
+		createFrontendDirectoryInAppDirectory();
```

Comment:
Since your global property value may be a full path that is not in the app directory, shouldn't we drop that part from the method name?